client/assets/sass/_forms.sass

```diff
@@ -1,3 +1,4 @@
+@import 'colors'
```

Comment:
this should be 'app_colors' (which is the only file that imports colors directly)